From 3f4fc73c2a0315c2da16dee7db0fcb2766f55d3c Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Tue, 29 Aug 2017 18:36:10 +0200
Subject: [PATCH 2/3] pr_gettext: stop printing extracted strings

They are written in the pot file already, no need to print them to
stderr (even though they are not errors).
---
 libgettext-ocaml/pr_gettext.ml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libgettext-ocaml/pr_gettext.ml b/libgettext-ocaml/pr_gettext.ml
index d78cf21..47d93e5 100644
--- a/libgettext-ocaml/pr_gettext.ml
+++ b/libgettext-ocaml/pr_gettext.ml
@@ -78,7 +78,6 @@ struct
       }
 
   let string_of_ocaml_string str =
-    prerr_endline str;
     Scanf.sscanf 
       (Printf.sprintf "\"%s\"" str)
       "%S"
-- 
2.13.2

